docs/src/test/scala/docs/http/scaladsl/SprayJsonPrettyMarshalSpec.scala

```diff
@@ -21,6 +20,15 @@ class SprayJsonPrettyMarshalSpec extends server.RoutingSpec {
       implicit val prettyPrintedItemFormat = jsonFormat2(PrettyPrintedItem)
     }
 
+    //#setup
+  }
+}
+
+class SprayJsonPrettyMarshalSpec extends server.RoutingSpec {
+
+  "spray-json example" in {
+    import SprayJsonPrettyMarshalSpec.setup._
+    //#example
     // use it wherever json (un)marshalling is needed
```

Comment:
I'd prefer something like this:

```scala
    // use it wherever json (un)marshalling is needed
    object PrettyJsonFormatSupport1 {
      import DefaultJsonProtocol._
      implicit val printer = PrettyPrinter
      implicit val prettyPrintedItemFormat = jsonFormat2(PrettyPrintedItem)
    }

    class MyJsonService extends Directives {
      import PrettyJsonFormatSupport1._
      import SprayJsonSupport._
```

which also follows the best practice not to inherit from `DefaultJsonProtocol`.